SamTxt.txt

```diff
@@ -0,0 +1 @@
+This is my txt file in KNguyen's repository 
```

Comment:
This sucks

Reply:
Please refrain from this kind of negativity.

![We_Don't_Do_That_Here](https://user-images.githubusercontent.com/14300728/74088456-4c8d5980-4a5c-11ea-9b8b-c2fcc0d4cc52.jpg)